universe/ValueRefs.cpp

```diff
@@ -249,6 +250,48 @@ namespace {
 }
 
 namespace ValueRef {
+
+template <>
+std::string ValueRef<std::string>::StringResult() const {
+    return Eval();
+}
+
+template <>
+std::string ValueRef<int>::StringResult() const {
```

Comment:
could probably make the `to_string` variant the default / templated implementation, and just have a specialization for `string` and `vector<string>`

Reply:
worked